pwm-fan.sh

```diff
@@ -471,6 +488,20 @@ while getopts 'c:C:d:D:fF:hl:m:p:s:t:T:' OPT; do
                 exit 1
             fi
             ;;
+        u)
+            THERMAL_ABS_THRESH_OFF="$OPTARG"
+            if [[ ! $THERMAL_ABS_THRESH_OFF =~ ^[0-4][0-9]?$ ]]; then
```

Comment:
```suggestion
            if [[ ! $THERMAL_ABS_THRESH_OFF =~ ^[0-4]?[0-9]$ ]]; then
```

otherwise, it won't match single digits `5, 6, 7, 8, 9`

Reply:
You are right, your suggestion fixes the issue. Then, the same problem is present at line 462 of the master branch.

Reply:
> I'll amend this after merging this PR into master. so, for now, just fix the part of the code I suggested.

author already addressed this in 6589b99, so nothing needs to be done after merging.